Add EIP: To The Moon—10 Minute Blocks (#6811)
ethereum/EIPs: EIPS/eip-asdf.md

---

> title: Ethereum To The Moon!

Reviewer: The title of this proposal isn't descriptive enough. Perhaps:

```suggestion
title: To The Moon—10 Minute Blocks
```

Or just:

```suggestion
title: Increase block time to 10 minutes
```

> description: Makes a minimal number of changes that allow Ethereum to be used on the moon

Reviewer: ```suggestion
description: Increases the block time to facilitate finality over cosmic distances
```

author: Pandapip1 (@Pandapip1)
discussions-to: https://ethereum-magicians.org/t/ethereum-to-the-moon/13633
status: Draft
type: Standards Track
category: Core
created: 2023-04-01
---

## Abstract

This EIP makes a minimal number of changes to allow Ethereum to be used on the moon and other potentially habitable bodies in Earth's solar system. It changes the time between blocks, the per-block validator reward, and the number of blocks per epoch.

## Motivation

> It is impossible for Ethereum to literally "go to the moon" due to a limitation in the protocol: the block length. Should validators attempt to validate on the surface of the moon, they would find that the ~1.25 second communication delay (caused by the speed of light) might cause issues with synchronization, considering the 12-second timer between blocks. The validators would eventually be ejected on the terrestrial chain after leaking. If however a substantial number of validators are displaced (think 1/3), they might follow their own fork and would eventually eject the terrestrial to finalize their own chain.

Reviewer: ```suggestion
It is impossible for today's Ethereum to literally "go to the moon" due to a limitation in the protocol: the block length. Should validators attempt to validate on the surface of the moon, they would find that the ~1.25 second communication delay (caused by the speed of light) might cause issues with synchronization, considering the 12-second timer between blocks. The validators would eventually be ejected on the terrestrial chain after leaking. If however a substantial number of validators are displaced (think 1/3), they might follow their own fork and would eventually eject the terrestrial to finalize their own chain.
```

## Specification

The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.

> * The time between blocks MUST be changed from 12 seconds to 10 minutes (50x longer).

Reviewer: ```suggestion
* The time between blocks MUST be changed from 12 seconds to 600 seconds (10 minutes).
```

* The per-block validator reward MUST be multiplied by 50
* The number of blocks per epoch MUST be reduced from 4 to 2

## Rationale

> * The block gas limit is multiplied by 50 to compensate for the time between blocks being multiplied by fifty.

Reviewer: ```suggestion
* The block gas limit is multiplied by fifty to compensate for the time between blocks being multiplied by fifty.
```

> * The per-block validator reward is also multiplied by 50 to copensate for the time between blocks being multiplied by fifty.

Reviewer: ```suggestion
* The per-block validator reward is also multiplied by fifty to compensate for the time between blocks being multiplied by fifty.
```
